config/src/main/java/org/springframework/security/config/annotation/web/builders/HttpSecurity.java

```diff
@@ -1324,11 +1324,10 @@ public HttpSecurity authorizeRequests(
 	 * 	&#064;Override
 	 * 	protected void configure(HttpSecurity http) throws Exception {
 	 * 		http
-	 * 		 	.authorizeHttpRequests((authorizeHttpRequests) ->
-	 * 		 		authorizeHttpRequests
-	 * 			 		.antMatchers(&quot;/**&quot;).hasRole(&quot;USER&quot;)
-	 * 			 		.antMatchers(&quot;/admin/**&quot;).hasRole(&quot;ADMIN&quot;)
-	 * 		 	);
+	 *     .authorizeHttpRequests()
```

Comment:
Did you mean to change the `authorizeHttpRequests()` JavaDoc instead? It looks like this got applied to `authorizeHttpRequests(Customizer)` instead.

Reply:
Sorry about that , got mixed